### PR TITLE
style(menu): make language toggle publicly visible

### PR DESCRIPTION
**Pull request by zushar**
Screenshot 2024-04-16 165926
Screenshot 2024-04-16 165939

# Description
<Please describe the suggested changes>

## screenshots
<paste here>

### CODE REVIEW

**Comment by zushar** [comment]
this is my fust pull 
סליחה על הטעיות אשמח לשמוע הערות

File: src/layout/sidebar/menu/LanguageToggle.tsx [hunk 1]
```diff
@@ -12,13 +11,13 @@ export const LanguageToggle = () => {
   }, 'he')
 
   return (
-    <EasterEgg code="english">
+    <>
```
Comment (NoamGaash) [inline src/layout/sidebar/menu/LanguageToggle.tsx:14]: this empty fragment can be removed

**Review by NoamGaash — APPROVED** [review]
Thanks! but don't we want to to anything regarding the design? I think the button looks like it's taken from another theme. If you want, please feel free to apply some CSS and adjust it
Thanks for your contribution and welcome to the bus!

**Comment by zushar** [comment]
i thought about moving it next to the icons on the top of the page and change it to the shape of a flag

**Comment by NoamGaash** [comment]
@zushar good idea :muscle:

**Pull request closed without merging**